apps/ex_nvr/lib/ex_nvr/pipeline/source/rtsp/connection_manager.ex

```diff
@@ -215,11 +216,12 @@ defmodule ExNVR.Pipeline.Source.RTSP.ConnectionManager do
            reconnect_delay: delay
          } = connection_status
        ) do
-    connection_status = %{connection_status | reconnect_attempt: attempt + 1}
+    new_delay = delay * :math.pow(2, attempt)
```

Comment:
I suggest you copy this algorithm from [here](https://github.com/ex-aws/ex_aws/blob/main/lib/ex_aws/request.ex#L212)
Just put the following `:base_backoff_in_ms` and `:max_backoff_in_ms` in module attributes. Put the base to 10 and the max to `:timer.minutes(2)`.